spotbugs/src/test/java/edu/umd/cs/findbugs/config/UserPreferencesTest.java

```diff
@@ -14,4 +18,44 @@ public void testClone() {
         Assert.assertEquals(sut, clone);
         Assert.assertEquals(sut.getClass(), clone.getClass());
     }
+
+    @Test
+    public void testReadEffortNoPrefix() throws IOException {
+        String testPrefsString = "effort=max";
+
+        UserPreferences userPrefs = UserPreferences.createDefaultUserPreferences();
+
+        userPrefs.read(new ByteArrayInputStream(testPrefsString.getBytes(StandardCharsets.ISO_8859_1)));
```

Comment:
Why ISO-8859-1 instead of UTF-8?

Reply:
According to https://docs.oracle.com/javase/8/docs/api/java/util/Properties.html#load-java.io.InputStream- the Properties are assumed to be in ISO 8859-1, so I used that as the encoding.